docs/pages/_error.js

```diff
@@ -206,41 +226,34 @@ function endsInNull(path) {
 // Simple remapping of renamed pages, similar to in deploy.sh but in some cases,
 // for reasons I'm not totally clear on, those redirects do not work
 const RENAMED_PAGES = {
-  '/versions/latest/introduction/project-lifecycle/':
-    '/versions/latest/introduction/managed-vs-bare/',
-  '/versions/latest/guides/': '/versions/latest/workflow/exploring-managed-workflow/',
+  '/introduction/project-lifecycle/': '/introduction/managed-vs-bare/',
+  '/guides/': '/workflow/exploring-managed-workflow/',
   '/versions/latest/sdk/': '/versions/latest/sdk/overview/',
-  '/versions/latest/guides/building-standalone-apps/':
-    '/versions/latest/distribution/building-standalone-apps/',
-  '/versions/latest/guides/genymotion/': '/versions/latest/workflow/android-studio-emulator/',
-  '/versions/latest/workflow/upgrading-expo/':
```

Comment:
Can we make sure old URIs such as https://docs.expo.io/versions/latest/workflow/upgrading-expo-sdk-walkthrough will still continue to work as redirects? Our documentation has many inbound links across the web (and quite possibly in internal documents of companies were we can't even see them), it would be a shame to lose them. [Cool URIs don't change.](https://www.w3.org/Provider/Style/URI)

Reply:
yep, already did this